easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-2.9.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'MakeCp'
+
+name = 'HiC-Pro'
+version = '2.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://nservant.github.io/HiC-Pro'
+description = """HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data) 
+ to the normalized contact maps."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/nservant/HiC-Pro/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    'b82f15747bce937c3337b8120ad331a4bc07bbed6708d1d38b7d6ff1047a7a81',  # v2.9.0.tar.gz
+    '7006902cff8367b6bf1ef8cabd4de26bdbfa842062a0250755ff703227e88f34',  # HiC-Pro.patch
+]
+
+exts_defaultclass = 'PythonPackage'
+
+patches = ['HiC-Pro.patch']
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('Pysam', '0.9.1.4', versionsuffix),
+    ('bx-python', '0.7.4', versionsuffix),
+    ('Bowtie2', '2.3.2'),
+    ('SAMtools', '1.3.1'),
+    ('R', '3.2.3')
+]
+
+exts_list = [
+    ('iced', '0.4.2', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/hiclib/iced/archive/'],
+        'checksums': ['6a199d713b1d8211c843afa8ec0caa83c640212470e735ebde3729c344537690'],
+    }),
+]
+
+skipsteps = ['configure']
+
+files_to_copy = [
+    ('scripts'),
+    (['bin/utils', 'bin/HiC-Pro'], 'bin'),
+    (['config-system.txt'], '')
+]
+
+modextrapaths = {
+    'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages'],
+    'SCRIPTS': ['scripts']
+}
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+sanity_check_paths = {
+    'files': ['bin/ice', 'config-system.txt', 'scripts/build_matrix', 'scripts/cutsite_trimming', 'bin/HiC-Pro'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages', 'scripts', 'bin/utils'],
+}
+
+postinstallcmds = [
```

Comment:
this should be moved up, below `files_to_copy` (order doesn't matter in an easyconfig file, but style-wise we try to keep easyconfig parameter more-or-less in order of execution)